Carbon.Core/Functions/Resolve-CFullPath.ps1

```diff
@@ -0,0 +1,46 @@
+
+function Resolve-CFullPath
+{
+    <#
+    .SYNOPSIS
+    Converts a relative path to an absolute path.
+
+    .DESCRIPTION
+    Unlike `Resolve-Path`, this function does not check whether the path exists.  It just converts relative paths to

```

Comment:
Maybe add a note that this function uses .NET's `[IO.Path]::GetFullPath(String)` static method?